parse_test.go

```diff
@@ -428,3 +429,30 @@ func TestParse_stdfs(t *testing.T) {
 		t.Errorf("foo: want %q, have %q", want, have)
 	}
 }
+
+func TestParse_shortSameCase(t *testing.T) {
+	t.Parallel()
+
+	newFS := func() *ff.FlagSet {
+		fs := ff.NewFlagSet(t.Name())
+		_ = fs.String('a', "apple", "", "foo string")
+		_ = fs.String('A', "apricot", "", "FOO string")
+		return fs
+	}
+	args := []string{"-a", "-A"}
+
+	t.Run("no env", func(t *testing.T) {
+		t.Parallel()
+		if err := ff.Parse(newFS(), args); err != nil {
+			t.Error(err)
+		}
+	})
+	t.Run("WithEnv", func(t *testing.T) {
+		t.Parallel()
+		if err := ff.Parse(newFS(), args, ff.WithEnvVars()); err == nil {
+			t.Error("wanted ErrDuplicateFlage, got nil")
+		} else if !errors.Is(err, ff.ErrDuplicateFlag) {
+			t.Errorf("wanted ErrDuplicateFlage, got %+v", err)
```

Comment:
Do you need the first if?


```suggestion
		if err := ff.Parse(newFS(), args, ff.WithEnvVars()); !errors.Is(err, ff.ErrDuplicateFlag) {
			t.Errorf("wanted ErrDuplicateFlag, got %+v", err)
```

I mean the %+v will report something for nil: `<nil>`

Reply:
Thanks